accepted/rfc-process.md

```diff
@@ -0,0 +1,109 @@
+
+*   **Coordination within a project**. As the BA grows, we hope and expect that projects will be actively developed by multiple organizations, rather than just by a “home” organization. While day-to-day activity can be handled through issues, pull requests, and regular meetings, having a dedicated RFC venue makes it easier to separate out discussions with far-ranging consequences that all project developers may have an interest in.
+
+# Proposal
+
+The design of this RFC process draws ideas from Rust’s [RFC](https://github.com/rust-lang/rfcs/) and [MCP](https://forge.rust-lang.org/compiler/mcp.html) processes, adapting to the BA and trying to keep things lightweight.
+
+## Stakeholders
+
+Each core BA project has a formal set of **stakeholders**. These are individuals, organized into groups by project and/or member organization. Stakeholders can block proposals, and conversely having explicit sign-off from at least one individual within each stakeholder group is a sufficient (but not necessary) condition for immediately accepting a proposal.
+
+The process for determining core BA projects and their stakeholder set will ultimately be defined by the Technical Steering Committee, once it is in place. Until then, the current BA Steering Committee will be responsible for creating a provisional stakeholder arrangement, as well as deciding whether to accept this RFC.
+
+## Structure and workflow
+
+### Creating and discussing an RFC
+
+*   We have a dedicated bytecodealliance/rfcs repo that houses _all_ RFCs for core BA projects, much like Rust’s rfcs repo that is shared between all Rust teams. 
+*   The rfcs repo will be structured similarly to the one in Rust:
+    *   A template markdown file laying out the format of RFCs, like [Rust’s](https://github.com/rust-lang/rfcs/blob/master/0000-template.md) but simplified.
+    *   A subdirectory holding the text of all accepted RFCs, like [Rust’s](https://github.com/rust-lang/rfcs/tree/master/text).
+*   New RFCs are submitted via pull request, and both technical and process discussion happens via the comment thread.
+*   The RFC is tagged with a **project label**, corresponding to the BA project it targets. This tagging informs tooling about the relevant stakeholder set.
+
+### Making a decision: merge or close
+
+*   When discussion has stabilized around the main points of contention, any stakeholder can make a **motion to finalize**, using a special comment syntax understood by tooling. This motion comes with a disposition: merge or close.
+*   In response to the motion to finalize, a bot will post a special comment with a **stakeholder checklist**. 
+    *   This list includes the GitHub handle for each individual stakeholder, organized into stakeholder groups. 
+    *   The individual who filed the motion to finalize is automatically checked off.
```

Comment:
You might mention, somewhere in this proposal, that we may be able to use Rust's rfcbot for this. (In general, I think it's important for proposals that assume certain technological support to be clear on whether that support is already mostly written or would need to be written.)